src/Database/Query.php

```diff
@@ -901,21 +808,16 @@ public function update($values = null, $where = null)
 	 * Fires a delete query
 	 *
 	 * @param mixed $where You can pass a where clause here or set it with ->where() before
-	 * @return bool
 	 */
-	public function delete($where = null)
+	public function delete(mixed $where = null): bool
```

Comment:
When `debug` enabled, `$this->execute()` method returns `array`. Is that important?